controlType:
  armJointPositionControl: false
  baseJointVelocityControl: true
  armJointPositionControl2: true
